django_netjsongraph/contextmanagers.py

```diff
@@ -11,5 +11,5 @@ def log_failure(action, obj):
     except Exception as e:
         msg = 'Failed to perform {0} on {1}'.format(action, obj.__repr__())
         logger.exception(msg)
-        print('{0}: {1}\nSee error log for more '
-              'information\n'.format(msg, e.__class__.__name__))
+        print(('{0}: {1}\nSee error log for more '
```

Comment:
I don't think we need this change, do we? 
Are you using some software to change python 2 code to python 3? (Which is allowed, don't worry, I only want to know what prompted this!)